requirements/extra-ml-requirements.txt

```diff
@@ -64,3 +64,5 @@ tenacity
 langchain
 # Required by mlflow.sentence_transformers
 sentence-transformers
+# Required by mlflow.keras_core
+keras-core
```

Comment:
We can test keras-core as we test keras.

Reply:
keras-core is still at 0.X, no backward compatibility is guaranteed. We can skip the cross-version testing for now I guess.